en/docs/insights/choreo-insights-api.md

```diff
@@ -4236,6 +4581,184 @@ Target-related errors.
 </tbody>
 </table>
 
+### GroupByField
+
+Represents an groupByField used in CustomReports.
```

Comment:
Suggestion:

"Represents a `groupByField` parameter used in `CustomReports`."

When you use a term in this format (e.g., groupByField), have it in code format and state what it is (e.g., `groupByField` parameter). Please do the same to `CustomReports` as well as all the similar terms in this document.

Reply:
ack